src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
not fixed:
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/4258541e-3d45-4121-a0f8-d610f46558d2)

again, instead of setting it in many components just set `box-sizing: border-box;` property in `*` selector, and remove everywhere else